### PR TITLE
Cuda 10 dirty fix

### PR DESCRIPTION
**Pull request by ddobreff**
Since we don't use macros defining CUDA arch, this should do for now.

### CODE REVIEW

**Comment by XhmikosR** [comment]
Yup :)

**Comment by ddobreff** [comment]
It does, using macros is even smarter :) Thats why I pushed it as dirty.

**Comment by StefanOberhumer** [comment]
> It does, using macros is even smarter :) Thats why I pushed it as dirty.

???
`list(APPEND ...` is a default cmake command not a macro

File: libethash-cuda/CMakeLists.txt [hunk 1]
```diff
@@ -24,7 +24,22 @@ else()
 		"-gencode arch=compute_61,code=sm_61"
 		"-gencode arch=compute_62,code=sm_62"
 		"-gencode arch=compute_70,code=sm_70"
+                "-gencode arch=compute_75,code=sm_75"
```
[the remark this answers is not in the document]
Comment (ddobreff) [reply libethash-cuda/CMakeLists.txt]: No, sm75 requires CUDA10 and will break 9.x compilation. I will redact it as of StefanOberhumer suggestion. It does look much better.

**Comment by ddobreff** [comment]
I meant to define gencode.. as macro and only use versions. Just like xmr-stak did it.

**Pull request merged**